scripts/build.js

```diff
@@ -0,0 +1,84 @@
+import chalk from "chalk";
+import fse from "fs-extra";
+
+const RECOMMENDATIONS_DIR = new URL("../recommendations", import.meta.url);
+const FEATURES_URL = new URL("../features.json", import.meta.url);
+
+const listRecommendations = async () => {
+  try {
+    const files = (await fse.readdir(RECOMMENDATIONS_DIR)).filter((file) =>
+      file.endsWith(".json")
+    );
+    console.log(chalk.blue(`Found ${files.length} recommendation files...`));
+    return files;
+  } catch (err) {
+    console.log(chalk.redBright(`Error reading recommendation dir: ${err}`));
+    throw err;
+  }
+};
+
+const loadFeatures = async () => {
+  try {
+    const features = await fse.readJson(FEATURES_URL);
+    console.log(
+      chalk.blue(`Found ${Object.keys(features).length} feature entries...`)
+    );
+    return features;
+  } catch (err) {
+    console.log(chalk.redBright(`Error loading features list: ${err}`));
+    throw err;
+  }
+};
+
+const buildRecommendationsObj = (files, recommendations) => {
+  const result = {};
+
+  console.log(chalk.blue(`Constructing recommendations object...`));
+
+  for (let i = 0; i < files.length; i++) {
+    const uuid = files[i].replace(".json", "");
+    result[uuid] = recommendations[i];
+  }
+
+  return result;
+};
+
+const loadRecommendations = async (files) => {
```

Comment:
We have 2 `loadRecommendations` methods defined here which is confusing. Can we rename one of them? Maybe the first one to `loadRecommendationFiles`?